### PR TITLE
V195 final fixes

### PR DESCRIPTION
**Pull request by leeping**

### CODE REVIEW

**Comment by leeping** [comment]
As of right now, the only test that isn't working is the Evaluator liquid bromine study.

**Comment by j-wags** [comment]
Looking into this now!

File: src/gmxio.py [hunk 1]
```diff
@@ -544,7 +544,7 @@ def setopts(self, **kwargs):
         if 'gmxsuffix' in kwargs:
             self.gmxsuffix = kwargs['gmxsuffix']
         else:
-            warn_once("The 'gmxsuffix' option were not provided; using default.")
+            warn_once("The 'gmxsuffix' option were not provided; using default (blank).")
```
Comment (j-wags) [inline src/gmxio.py:547]: (not blocking) Is the (blank) intentional?

**Review by j-wags — APPROVED** [review]
Looks good to me!

**Comment by leeping** [comment]
Yes, it’s intentional. Thanks!

- Lee-Ping

**Comment by leeping** [comment]
Thanks - I'll merge it now!

**Pull request merged**